lightning/src/util/config.rs

```diff
@@ -128,6 +135,24 @@ pub struct ChannelHandshakeLimits {
 	///
 	/// Default value: 144, or roughly one day and only applies to outbound channels.
 	pub max_minimum_depth: u32,
+	/// Whether we implicitly trust our own funding transactions to not be double-spent.
+	///
+	/// If this is set, we assume that our own funding transactions are *never* double-spent, and
+	/// thus we can trust them without any confirmations. This is generally a reasonable
+	/// assumption, given we're the only ones who could ever double-spend it (assuming we have sole
+	/// control of the signing keys).
+	///
+	/// You may wish to un-set this if you allow the user to (or do in an automated fashion)
+	/// double-spend the funding transaction with an alternative channel open to RBF a channel
+	/// opening transaction.
```

Comment:
If a 0conf user is willing to fee-bump, should we advice them to CPFP the funding transaction through a change output, if any ?

Reply:
Sure, that seems implied, no? You can of course RBF a funding transaction by simply opening a new channel and closing the original, but that requires non-0-conf.

Reply:
Hmmm you might be willing to use CPFP to get an instant usage effect that you loose with RBF as the replacement transaction invalids the commitment transactions.